parity/account.rs

```diff
@@ -86,3 +94,17 @@ fn import(i: ImportAccounts) -> Result<String, String> {
 	}
 	Ok(format!("{}", imported))
 }
+
+fn import_geth(i: ImportFromGethAccounts) -> Result<String, String> {
+	use std::io::ErrorKind;
+	use ethcore::ethstore::Error;
+
+	let dir = Box::new(try!(keys_dir(i.to)));
+	let secret_store = EthStore::open(dir).unwrap();
```

Comment:
Copy-pasted this line from other functions above (new, list) :) Just tried to mimic the code-style from previous contributors. Ok - will fix this for these functions too